scripts/variantstore/wdl/GvsCreateFilterSet.wdl

```diff
@@ -343,6 +364,11 @@ task UploadFilterSetToBQ {
 
         export GATK_LOCAL_JAR=~{default="/root/gatk.jar" gatk_override}
 
+        if [ ~{has_service_account_file} = 'true' ]; then
+          gcloud auth activate-service-account --key-file='~{service_account_json}'
```

Comment:
might be good to set the environment variable here as well... even if it's not strictly necessary it keeps out "service account auth" stanza the same everywhere. so our future selves won't wonder why they are different

Reply:
great idea